### PR TITLE
use unified dynamic script for amplitude instrumentation

### PR DESCRIPTION
**Pull request by bgiori**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: overrides/partials/integrations/analytics/amplitude.html [hunk 1]
```diff
@@ -1,11 +1,22 @@
+<!-- Unified Dynamic Script: contains analytics, experiment, session replay & auto-capture -->
+<script src="https://cdn.amplitude.com/script/e7d181e4593e9f96.js"></script>
+<!-- Opt the user out before consent is approved. -->
+<script type="text/javascript">
```
Comment (izaaz) [inline overrides/partials/integrations/analytics/amplitude.html:4]: Should this be above the script tag?

**Review by markzegarelli — APPROVED** [review]

File: overrides/partials/integrations/analytics/amplitude.html [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+    window.amplitude.init('c1817405c2f578878637c55704d4a153', {
+        defaultEventTracking: true,
+        identityStorage: 'localStorage',
+        optOut: true,
```
Comment (izaaz) [inline overrides/partials/integrations/analytics/amplitude.html:8]: And why optOut = true?

**Review by zhukaihan — APPROVED** [review]

File: overrides/partials/integrations/analytics/amplitude.html (hunk 1 above)
Comment (zhukaihan) [reply overrides/partials/integrations/analytics/amplitude.html:4]: The `amplitude` and plugins found on `window` are stubbed by the script tag.

Comment (izaaz) [reply overrides/partials/integrations/analytics/amplitude.html:4]: ah nvm. makes sense.

File: overrides/partials/integrations/analytics/amplitude.html [hunk 3]
```diff
@@ -1,11 +1,22 @@
+<!-- Unified Dynamic Script: contains analytics, experiment, session replay & auto-capture -->
+<script src="https://cdn.amplitude.com/script/e7d181e4593e9f96.js"></script>
+<!-- Opt the user out before consent is approved. -->
+<script type="text/javascript">
+    window.amplitude.init('c1817405c2f578878637c55704d4a153', {
+        defaultEventTracking: true,
```
Comment (zhukaihan) [inline overrides/partials/integrations/analytics/amplitude.html:6]: Didn't see this option in original config, is the default value true?

Comment (bgiori) [reply overrides/partials/integrations/analytics/amplitude.html:7]: Yeah default value is true, it actually logs a warning to the console, so this is just to squelch that warning

File: overrides/partials/integrations/analytics/amplitude.html (hunk 2 above)
Comment (bgiori) [reply overrides/partials/integrations/analytics/amplitude.html:9]: We have a tracking opt-in popup in the dev docs. If the customer does not opt-in we dont want to track them.

**Pull request merged**